libs/dyn/mapping.go

```diff
@@ -144,6 +145,7 @@ func (m Mapping) Clone() Mapping {
 // Merge merges the key-value pairs from another Mapping into the current Mapping.
 func (m *Mapping) Merge(n Mapping) {
 	for _, p := range n.pairs {
-		m.Set(p.Key, p.Value)
+		// QQQ log error or panic?
+		_ = m.Set(p.Key, p.Value)
```

Comment:
Same; the mapping can only be constructed with string keys.

Reply:
Removed the comment and added //nolint:errcheck annotation.